xun/init/cookiecutter-workflow/{{cookiecutter.project_name}}/__main__.py

```diff
@@ -0,0 +1,59 @@
+"""
+This module file is the entrypoint in case the project package is run with
+`python -m {{cookiecutter.project_name}}`.
+For more info, read:
+    - https://docs.python.org/3/using/cmdline.html#cmdoption-m
+"""
+
+from .{{cookiecutter.project_name}} import capitalize
+import argparse
+import logging
+import xun
+
+
+logging.basicConfig(level=logging.WARNING)
+logging.getLogger('xun').setLevel(logging.DEBUG)
+
+
+"""
+Driver and store setup
+For driver, choose between sequential or dask
+For store, choose between disk, memory, redis or sftp
+"""
+driver = xun.functions.driver.Sequential()
+store = xun.functions.store.Memory()
+
+
+def main():
+    """
+    If this workflow is to be executable, this is where the setup for the
+    driver and the store should go, along with the entrypoint function(s).
+
+    The argument parser is optional and can be included in case there are
+    arguments that should be possible to pass from the command line.
+    """
+    parser = argparse.ArgumentParser(
+        description='Greet the user with a personalized message!',
+    )
+    parser.add_argument(
+        '--name',
+        help=(
+            'a name to be greeted by the program (example: Jane Doe)'
+        ),
+        default='World'
+    )
+
+    args = parser.parse_args()
+
+    @xun.function()
+    def entry_point_workflow(name):
+        print(f'Hello {capitalized_name}')
+        with ...:
+            """Set up your workflow here"""
+            capitalized_name = capitalize(name)
+
+    entry_point_workflow.blueprint(args.name).run(driver=driver, store=store)
```

Comment:
Maybe we should drop the parser? Just to not encourage users to write general programs

Reply:
Okey, that is reasonable. The example then makes a little less sense, with regards to user input, but still shows the concept i guess.